### PR TITLE
adding updated pictures for spanish field papers page

### PR DESCRIPTION
**Pull request by d3netxer**

### CODE REVIEW

**Comment by Nick-Tallguy** [comment]
Hi Tom, @d3netxer and thanks for your work on this - looks pretty good.  

I use a [staging site just to check on things and you can view how your updates will look here](http://nick-tallguy.github.io/es/mobile-mapping/field-papers/)  

Some of the images in your pull request are slightly on the long side - a maximum size of 520px longest edge, are used on LearnOSM - [see wiki](https://github.com/hotosm/learnosm/wiki/Images-guidelines). Could you reduce them please, or I can do them if needed.

GitHub is in the process of changing from rdiscount to kramdown, and it is producing some slightly unexpected results for us - see #498 for my comments. The staging site is using kramdown. Your pull request has highlighted another problem in that the numbered lists are now not working the way they should do. It's nothing that you've done - but if you look on the staging site you will see that there is paragraph 1, followed by paragraph 1, and then paragraph 1, before it finally settles down and moves on to point 2. I haven't investigated to see what has caused this strange behavior and can only put it down to the change to kramdown. 

Before I upload any changes to the main site I run them locally using 'jekyll serve', and the same numbering behavior can be seen there. If you copy the [_config.yml](https://github.com/Nick-Tallguy/Nick-Tallguy.github.io/blob/master/_config.yml) from my staging site & use it in your files you will see the effect locally.

So, perhaps you would consider making the changes to <https://github.com/Nick-Tallguy/Nick-Tallguy.github.io/blob/master/_posts/es/mobile-mapping/0900-12-23-papers.md> at the same time, so that the numbered lists display correctly as well. Don't worry if you can't, I'll work on it when I can. 

## In summary, please can you;  
1.  Reduce image sizes, and  
2.  Consider amending the main file so that the numbering works correctly.

& thanks again, you've made a very good job of this update.

Regards

Nick

**Comment by d3netxer** [comment]
I can work on this at some point, but probably when I get back from my trip after 17 Feb

**Comment by d3netxer** [comment]
@Nick-Tallguy good idea, I can work on the staging site, please give me access. You will see I will make a commit on it; where I have fixed the numbering issue. I was reading this http://stackoverflow.com/questions/18088955/markdown-continue-numbered-list and it turns out that an indent will make the parser know that the img belongs within the list and won't break the numbering.

About the img size, I created a new issue, as I prefer a larger img size for screenshots
